extension/install.rdf

```diff
@@ -4,7 +4,7 @@
 
     <em:id>personas@christopher.beard</em:id>
     <em:name>Personas Plus</em:name>
-    <em:version>1.7.6</em:version>
+    <em:version>1.7.6.1</em:version>
```

Comment:
Actually, let's not merge code changes with version bumping. You can just remove that change entirely and I will do the bump, tag the rev etc.